artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/impl/PagingManagerImpl.java

```diff
@@ -353,7 +353,7 @@ public void run() {
    }
 
    public void debug() {
-      logger.info("size = " + globalSizeBytes + " bytes, messages = " + numberOfMessages);
```

Comment:
Please.. take this one out.. it's just a debug.. never used in real life.

Reply:
yeap, ok